pypeit/specobjs.py

```diff
@@ -195,6 +299,7 @@ def append_neg(self, sobjs_neg):
         sobjs_neg.sign = -1.0
         if sobjs_neg[0].PYPELINE == 'Echelle':
             sobjs_neg.ECH_OBJID = -1*sobjs_neg.ECH_OBJID
+            sobjs_neg.OBJID = -1*sobjs_neg.OBJID
```

Comment:
Does `-sobjs_neg.OBJID` not work (i.e., instead of `-1*sobjs_neg.OBJID`)?

Reply:
Changed.